developer/src/test/auto/keyboard-info/test-invalid-language-code.keyboard_info

```diff
@@ -1,6 +1,17 @@
 {
+    "id": "test",
```

Comment:
Should this id be `test_invalid_language_code`? (and use underscore instead of dashes)

Reply:
The id doesn't matter for this test.